lib/generators/manageiq/plugin/templates/config/secrets.defaults.yml

```diff
@@ -0,0 +1,7 @@
+---
+test:
+  <%= plugin_short_name %>_defaults: &<%= plugin_short_name %>_defaults
+    <%= plugin_short_name %>_username: username
```

Comment:
did you want to add hostname here too?  It's in a lot of the ones I've seen so far.

Reply:
Yeah only reason I didn't is because most cloud managers don't have a hostname, but everyone has username/password type credentials.